tests/ui/eta_nostd.rs

```diff
@@ -0,0 +1,10 @@
+// #![warn(clippy::redundant_closure)]
```

Comment:
Leaving it explicitly is better in case it is no longer active by default at some point.

```suggestion
#![warn(clippy::redundant_closure)]
```

Reply:
I forgot to put back it... Thanks, I fixed it.